helpers/index.js

```diff
@@ -0,0 +1 @@
+module.exports.colorProfileMapper = require("./colorProfileMapper");
```

Comment:
can we just keep adding `module.exports.` to this file as we create new helpers?

Reply:
:tomato: 🍅 Maybe we can structure it like we do here: https://git.fullscript.io/developers/hw-admin/-/blob/staging/babel/plugins/TsToSass/helpers/index.js